README.md

```diff
@@ -71,6 +71,14 @@ See [related documentation](./guides/alerting/alerting.md) for more information.
 
 **Trento** can be installed in various ways, depending on your needs and the requirements of its different components. Please refer to the specific [Agent](https://github.com/trento-project/agent) documentation for more information.
 
+## Installing Trento manually
+
+For manual instructions on installing Trento, please refer to our [Installation of Trento](https://github.com/trento-project/docs/blob/main/guides/manual-installation.md) guide.
+
+## Installing Trento using Ansible (WIP)
```

Comment:
I would remove the `WIP` reference.
The ansible playbook is "finished" so to say.